.github/workflows/cd.yml

```diff
@@ -64,17 +64,21 @@ jobs:
 
   deploy-k8s-service:
     name: Deploy to Kubernetes
+    runs-on: ubuntu-22.04
     needs:
       - tag
       - build-push-docker
-    uses: bitwarden/devops/.github/workflows/_deploy_k8s_service.yml@b9c4c99a4cb0330465af8b30c9a7a4e8202dec25
-    with:
-      image-names: billing-relay
-      environment: Mothership
-      overlay: mship-cloud-netnew
-      repository: billing-relay
-      tag: ${{ needs.tag.outputs.git-tag }}
-    secrets: inherit
```

Comment:
❓ Any concern with this not being here? We'd need to set environment variables.

Reply:
Are you referring to `secrets: inherit`? I don't think it needs any additional secrets. The ones in the `devops` repository should work.